recipe/meta.yaml

```
@@ -8,37 +8,50 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 46d3fd2016cd5b842283d547877a02c418a0fe9aa7a6b0ae344115a2c990fef4
+  patches:
+    - patches/0001-rm-test-jsonpath-py.patch
+    - patches/0002-safe-eval.patch
 
 build:
-  noarch: python
   number: 0
```

```suggestion
  number: 1
```
I'm not sure if this is necessary nor how to verify what already exists on the Snowflake channel, but if the version remains the same, should the build number be bumped up or do things work a little differently if not being uploaded to defaults?

---

Also, I missed why we're reverting to 0.82, can you please clarify?

---

`jsonpath` is new to us.  We had the feedstock in AnacondaRecipes but never published anything.  I could have kept the build number but... 🤷 

I've updated the description for 0.82.

---

It may be good moving forward to put "Initial build" either in the PR title or description for all the new folks joining the officially published group. Lol

---

That's a fair point, especially for those not coming from reading Jira, I'll see if I can tweak my code... 🤔